ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -35,6 +35,10 @@ echo "TESTING_RELEASE_IMAGE: ${TESTING_RELEASE_IMAGE}"
 dir=$(mktemp -d)
 pushd "${dir}"
 cp ${CLUSTER_PROFILE_DIR}/pull-secret pull-secret
+echo "OC Version:"
+export PATH=${CLI_DIR}:$PATH
```

Comment:
I think once we did https://github.com/openshift/release/pull/47531/files#diff-7b3301dd584bc6ffcff00864d1cf414155468b7bd6691fae1c7e633c8d393996L35, we do not need to do anything here, right?